app/views/pages/live.html.haml

```diff
@@ -0,0 +1,77 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'live.css'
+
+= render "layouts/hackathon_manager/flashes"
+= render "layouts/nav"
+
+.section#timer
+  .timer__section
+    %h2.text-center Timer
+    .timer__column__container
+      .timer__left
+        %h4.text-center Timer #1
+        .timer__container
+          %p.text-center{:id => "timer_countdown"}
+          :javascript
+            var x = setInterval(function() {
+            var countDownDate = new Date("Feb 11 2019 15:00:00").getTime();
+            var now = new Date().getTime();
+            var untilDate = countDownDate - now;
+            var hours = Math.floor((untilDate%(1000*60*60*24))/(1000*60*60));
+            var minutes = Math.floor((untilDate % (1000*60*60))/(1000*60));
+            var seconds = Math.floor((untilDate % (1000*60))/1000);
+            document.getElementById("timer_countdown").innerHTML = hours + "h "+minutes+"m "+seconds+"s ";
+            if(untilDate < 0){
+            clearInterval(x);
+            document.getElementById("timer_countdown").innerHTML = "00h 00m 00s";
+            }
+            }, 1000);
```

Comment:
Could you indent each line that is inside curly braces? Both for the function & `if` contents. Makes the code easier to read / conditional `if` logic stand out